app/src/sharedTest/java/org/oppia/android/app/recyclerview/BindableAdapterTest.kt

```diff
@@ -449,8 +418,8 @@ class BindableAdapterTest {
 
   private fun createSingleViewTypeNoDataBindingBindableAdapter():
     BindableAdapter<BindableAdapterTestDataModel> {
-      return SingleTypeBuilder
-        .newBuilder<BindableAdapterTestDataModel>()
+      return BindableAdapter.SingleTypeBuilder.Factory
+        .create<BindableAdapterTestDataModel>()
         .registerViewBinder(
           inflateView = this::inflateTextViewForStringWithoutDataBinding,
           bindView = this::bindTextViewForStringWithoutDataBinding
```

Comment:
Why this is showing an error to import `create`?
![Screenshot from 2021-03-06 14-56-31](https://user-images.githubusercontent.com/54636525/110202038-48648300-7e8c-11eb-86c9-8a60cd9b05c0.png)

I tried doing `Invalidate Caches and Restart` several times but it is not getting removed.

Reply:
Don't we need to inject the factory rather than trying to call it as if it's static/part of a companion object?

Reply:
Done.